.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      k8s.io:
+        patterns:
+          - "k8s.io/client-go"
```

Comment:
_groups_ are for when you have multiple dependencies you want updated together rather than each on their own. Since there is only one, you can remove the whole _groups_ section.